build/__dotNetCoreSdkVersions.sh

```diff
@@ -3,7 +3,7 @@
 DOT_NET_CORE_11_SDK_VERSION='1.1.14'
 DOT_NET_CORE_21_SDK_VERSION='2.1.818'
 DOT_NET_CORE_22_SDK_VERSION='2.2.207'
-DOT_NET_CORE_30_SDK_VERSION='3.0.103'
+DOT_NET_CORE_30_SDK_VERSION='3.1.417'
```

Comment:
Looks that we're updating dotnet 3.0 version constant here, we need to update 3.1 version constant  instead.

Reply:
Updated. Thanks Stella!